powerquery-docs/Connectors/DatabricksCloud.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
looking at the other connectors (e.g., BigQuery) they named their markdown file after their connector (i.e.,  GoogleBigQuery.md)
I think we should follow the same naming pattern: `DatabricksCloud.md` -> `Databricks.md`
any reason we have the Cloud suffix?

Reply:
Just habit/muscle memory. The name was "Databricks Cloud" back in the day: https://databricks.com/company/newsroom/press-releases/databricks-announces-general-availability-cloud-platform-dramatically-simplifies-big-data-processing 

But I have no problem paring it down.